Makefile

```diff
@@ -4,9 +4,13 @@ NAME=ybm
 VERSION=0.1.0-dev
 OS_ARCH=darwin_arm64
 BINARY=terraform-provider-${NAME}
+export GOPRIVATE := github.com/yugabyte
 
 default: install
 
+vet:
```

Comment:
did you rebase your branch ??

Reply:
This is the Terraform provider - it never had a vet command in the `Makefile`.

Reply:
Oh... that;s why the code was a bit different lol Sorry